### PR TITLE
chore: Disable CGO by default

### LINKED ISSUE #80: [feature] Improve the performance when saving/exporting the model
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind feature

**What happened**:

```
Using config file: /home/gaocegege/.ormb/config.json
Using /home/gaocegege/.ormb as the root path
ref:       localhost:5000/machinelearning/onnx-model:v1
digest:    a575cb4d62801c295caea441b2e431867148a82718c53551425c8beac8fcaf99
size:      28.9 MiB
format:    ONNX
v1: saved
```

Currently, Saving an 30M ONNX example takes about 30s.

```
Showing nodes accounting for 43.26s, 96.37% of 44.89s total
Dropped 93 nodes (cum <= 0.22s)
Showing top 10 nodes out of 50
      flat  flat%   sum%        cum   cum%
    28.96s 64.51% 64.51%     28.96s 64.51%  runtime._ExternalCode
     4.61s 10.27% 74.78%      4.61s 10.27%  racecall
     3.90s  8.69% 83.47%     13.36s 29.76%  compress/flate.(*compressor).deflate
     2.04s  4.54% 88.02%      2.04s  4.54%  racecalladdr
     1.33s  2.96% 90.98%      1.33s  2.96%  runtime.raceread
     0.73s  1.63% 92.60%      1.85s  4.12%  compress/flate.(*huffmanBitWriter).writeCode
     0.71s  1.58% 94.19%      1.03s  2.29%  compress/flate.(*compressor).findMatch
     0.35s  0.78% 94.97%      0.71s  1.58%  compress/flate.hash4 (inline)
     0.34s  0.76% 95.72%      0.34s  0.76%  runtime.racewrite
     0.29s  0.65% 96.37%      0.29s  0.65%  crypto/sha256.block

```

We should optimize it.

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #80 

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -44,6 +44,9 @@ REGISTRY ?= cleveross
 # Container registry for base images.
 BASE_REGISTRY ?= docker.io
 
+# Disable CGO by default.
+CGO_ENABLED ?= 0
```
Comment (simon-cj) [inline Makefile:48]: Maybe gorelease.yml need add this flag

Comment (gaocegege) [reply Makefile:48]: SGTM

**Comment by gaocegege** [comment]
Comments are addressed, PTAL

**Pull request merged**